src/components/ListItem.js

```diff
@@ -52,7 +57,8 @@ function ListItem({ data: movie, listDetails: list }) {
     }
   }
 
-  function toggleWatched() {
+  function toggleWatched(e) {
```

Comment:
This doesn't feel like much of an improvement to me. Having the drop down show "unwatched" or "watched"  is busy looking and redundant with its tab. 

Also, functionality-wise the user isn't really making a selection.  It's always a one or the other decision,  making toggle or check are more true to the behavior.

I'm not totally against a toggle though, and think it would probably look nicer than a check, but.. is the action really a toggle? Can a movie be unwatched?

My vote goes to improving the check but if you wanna go with a toggle I'd roll with it.

Reply:
The checkbox just felt very unclear to me. There's no label associated with it, no actions listed. There's nothing to actually indicate to the user that what they are doing is changing its state to watched or unwatched. 

If you think the select is too busy looking, I would suggest we add essentially a title above the column that the checkboxes appear in. 

And while you can't literally 'unwatch' a movie, you can realize it shouldn't be in your watched list and move it back to unwatched.

Reply:
Agree that the checkbox isnt clear, I was thinking about a title also and that this is really a table but don't think we have time to make that happen.

maybe the toggle is the path of least resistance and what we're looking for in terms of being able to move from being watched/unwatched ? unless we mock a table header?

Reply:
I think that a table is ultimately what would work best, but I think that's definitely a stretch goal. the tooltip I added should do for now. A table would allow us to have an actions drop down so you can delete or recategorize multiple movies at once.

Reply:
yup the tooltip seems to provide enough clarity for now, nice work all around